.changelog/1395.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+config: Add ability write configuration values as files rather than environment variables.
```

Comment:
Juuuust sliding in a little changelog fixup :disappears:
```suggestion
config: Add ability to write configuration values as files rather than environment variables.
```